app/layout.tsx

```diff
@@ -13,7 +13,7 @@ const manrope = Manrope({ subsets: ['latin'] });
 export const metadata: Metadata = {
   metadataBase: new URL(APP_URL),
   title: `${APP_NAME} - Where predictions meet AI agents`,
-  description: 'Bet now on multiple markets in gnosis chain.',
+  description: 'Bet on prediciton markets in Gnosis Chain, powered by AI.',
```

Comment:
powered by AI is a bit misleading, since it's not powered by AI, It's just AI aware at this point (showing AI agents). We'll show research, and models used by agents to explain the actions.

```suggestion
  description: 'Bet on prediciton markets in Gnosis Chain and follow AI agents bets and reasoning.',
```

Something like this.

Reply:
We are also showing markets created with AI. What about adding some mystery to what AI does so users click on it?

Reply:
I would prefer something like `enhanced by AI`, powered seems to strong for this case.